encoding/thrift_request_test.go

```diff
@@ -243,3 +243,20 @@ func TestFindMethod(t *testing.T) {
 		}
 	}
 }
+
+func TestWithoutEnvelopes(t *testing.T) {
+	serializer, err := NewThrift(validThrift, "Simple::foo")
+	require.NoError(t, err, "Failed to create serializer")
+
+	enveloped, err := serializer.Request([]byte("{}"))
+	require.NoError(t, err, "serialize enveloped failed")
+
+	serializer = serializer.(thriftSerializer).WithoutEnvelopes()
+	unenveloped, err := serializer.Request([]byte("{}"))
+	require.NoError(t, err, "serialize unenveloped failed")
```

Comment:
Is it worth verifying the serialized contents?

Unenveloped empty struct is just a single byte: `0x00`. Enveloped will be,

```
0x80, 0x01, 0x00, 0x01, // version | type = 1 | call
0x00, 0x00, 0x00, 0x03, 'f', 'o', 'o', // "foo"
0x00, 0x00, 0x00, 0x00, // whatever seqID you're using
0x00, // empty struct
```